<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="lw_operations"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="b_add">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_up">
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_down">
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_load">
         <property name="text">
          <string>Load...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_save">
         <property name="text">
          <string>Save...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="lw_queue"/>
       </item>
       <item>
        <widget class="QStackedWidget" name="stack">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <widget class="QWidget" name=""/>
         <widget class="QWidget" name="page_4"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="le_description"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
